_data/twenty_17/teams.yml

```diff
@@ -2,14 +2,19 @@
 - title: Thomas Wiecki
   name: Thomas-Wiecki
   subtitle: Executive Chair
-  twitter:
-  github:
-  web:
-  img: Pymc4.png
-  thumbnail: Pymc4.png
+  twitter: twiecki
+  github: twiecki
+  web: https://twiecki.io
+  img: twiecki.jpg
+  thumbnail: twiecki.jpg
   alt: Thomas Wiecki
-  topic:
-  description:
+  topic: Advocate for Bayesian methods and probabilistic programming
+  description: 'Thomas discovered the power of Bayesian statistics during his PhD at Brown University
```

Comment:
Looks like the apostrophe opens here but does not close at the end of the bio, not completely sure but I think this will be an invalid yaml

Reply:
Fixed.